ykman-gui/qml/PivManager.qml

```diff
@@ -0,0 +1,491 @@
+import QtQuick 2.5
+import QtQuick.Dialogs 1.2
+import QtQuick.Controls 1.4
+import QtQuick.Layouts 1.1
+import QtQuick.Window 2.0
+
+DefaultDialog {
+
+    height: calculateHeight()
+    minimumHeight: calculateHeight()
+    minimumWidth: calculateWidth()
+    width: minimumWidth
+    title: qsTr("PIV Manager")
```

Comment:
I don't think we should call this "PIV Manager", too confusing since it's another tool. What about just PIV Configuration or similar?

Reply:
Sure. Do you mind if the root component still has the ID `pivManager`, or should I change that too?